starboard/shared/starboard/player/filter/testing/test_util.cc

```diff
@@ -181,14 +181,32 @@ std::vector<VideoTestParam> GetSupportedVideoTests() {
       const auto& video_stream_info = dmp_reader.video_stream_info();
       const std::string video_mime = dmp_reader.video_mime_type();
       const MimeType video_mime_type(video_mime.c_str());
-      if (SbMediaIsVideoSupported(
-              dmp_reader.video_codec(),
-              video_mime.size() > 0 ? &video_mime_type : nullptr, -1, -1, 8,
-              kSbMediaPrimaryIdUnspecified, kSbMediaTransferIdUnspecified,
-              kSbMediaMatrixIdUnspecified, video_stream_info.frame_width,
-              video_stream_info.frame_height, dmp_reader.video_bitrate(),
-              dmp_reader.video_fps(), false)) {
-        test_params.push_back(std::make_tuple(filename, output_mode));
+      // SbMediaIsVideoSupported may return false for gpu based decoder that in
+      // fact supports av1 or/and vp9 because the system can make async
+      // initialization at startup.
+      // To minimize probability of false negative we check result few times
```

Comment:
This comment may be confusing as it only affects one platform, while this file is used by every platform. At the same time, this isn't an internal file so we're limited in what we can say. It may be better to explore creating an internal test_util.cc for this case.

Reply:
We can mention internal ports like PS4 and Xbox, they are mentioned elsewhere in the code. I'd also not split the code unless it's really necessary, default to things in the open.

Reply:
If to expand SbMediaIsVideoSupported interface as I offered above there won't be necessary in any comments concerning delay at all